package.json

```diff
@@ -68,7 +68,7 @@
 			"module": "./lib/esm-bundle.mjs",
 			"import": "./lib/esm-bundle.mjs",
 			"require": "./lib/cjs-bundle.cjs",
-			"types": "./lib/types-bundle.mjs",
+			"types": "./lib/types-bundle.d.ts"
```

Comment:
I would love to have a comma at the end of this line, npm really does not like it 😂